apps/slack/frontend/src/constants.ts

```diff
@@ -5,5 +5,5 @@ export const makeOAuthURL = (spaceId: string, environmentId: string) => {
   const redirectUri = encodeURIComponent(
     `${BACKEND_BASE_URL}/oauth?spaceId=${spaceId}&environmentId=${environmentId}`
   );
-  return `https://slack.com/oauth/v2/authorize?client_id=${CLIENT_ID}&scope=chat:write,channels:read,team:read&redirect_uri=${redirectUri}`;
+  return `https://slack.com/oauth/v2/authorize?client_id=${CLIENT_ID}&scope=chat:write,channels:read,groups:read,team:read&redirect_uri=${redirectUri}`;
```

Comment:
I think ultimately this is not a necessary change as we determined the scope change needs to occur in the Bot settings.